kitsune/search/urls.py

```diff
@@ -1,10 +1,7 @@
 from django.conf.urls import url
 
-from kitsune.search import views
 from kitsune.search.v2 import views as v2_views
 
 urlpatterns = [
     url(r"^$", v2_views.simple_search, name="search"),
-    url(r"^/xml$", views.opensearch_plugin, name="search.plugin"),
```

Comment:
I think it may be worth keeping the opensearch xml, just without the suggestions piece.

I think all we need to do in `kitsune/search/jinja2/search/plugin.html` is remove the `<Url type="application/x-suggestions+json" />` element, and it'll be compatible with search v2.